packages/xo-web/src/xo-app/home/index.js

```diff
@@ -223,6 +223,11 @@ const OPTIONS = {
         sortBy: 'startTime',
         sortOrder: 'desc',
       },
+      {
+        labelId: 'homeSortByInstallTime',
+        sortBy: 'installTime',
```

Comment:
IMO, `null` start times should be at the end. Not only because they're not relevant, but also because it most likely means that they have been created a long time ago anyway.

Reply:
I thought by sorting by ```desc``` the nulls would be at the end but weirdly they are at the beginning of the list

Reply:
They are likely consider either greater or lesser than all numbers.

Which means they are either at the start or the end depending on your sort order.

And yes, you need to put them at the end as per the specification for this feature.